packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -211,12 +211,12 @@ export class VaRadio {
         aria-invalid={error ? 'true' : 'false'}
         aria-label={ariaLabel}
       >
-        <legend>
+        <legend part="legend">
```

Comment:
I also added the `part` property to the `va-radio` as well as the `va-radio-option` because in our case, we want to make the text in the legend bold and a larger font - [primary phone design for Supplemental Claims](https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/p/3EA2CBFB-0365-40BA-B9ED-77C233A93C6B/canvas#Version)

<img width="365" alt="Supplemental claims primary phone page design with a large & bold legend of 'what is your primary phone number?'" src="https://user-images.githubusercontent.com/136959/200050293-9e9d6019-9855-423f-9f1d-a4a0d07237b0.png">

Reply:
This legend `part` attribute was added by #582